### PR TITLE
fix typos in governance.md

### PR DESCRIPTION
**Pull request by 404Nikhil**

### CODE REVIEW

**Comment by 404Nikhil** [comment]
@leecalcote please review this PR

File: GOVERNANCE.md [hunk 1]
```diff
@@ -25,11 +25,11 @@ The Meshery project consists of multiple repositories. Each repository is subjec
 
 ### Maintainers
 
-Maintainers have write access to repositories. The current maintainers of a repo can be found in [MAINTAINERS.md](./MAINTAINERS.md) file of the repo.
+Maintainers have written access to repositories. The current maintainers of a repo can be found in [MAINTAINERS.md](./MAINTAINERS.md) file of the repo.
```
Comment (leecalcote) [inline GOVERNANCE.md:28]: Revert this.

File: GOVERNANCE.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ Anyone can become a Meshery contributor simply by contributing to the project, w
 
 ## Contributors
 
-GitHub organization members are people who have `triage` access to all repos in the organization. Community members who wish to become members of the organization should meet the following requirements, which are open to the discretion of the steering committee:
+GitHub's organization members are people who have `triage` access to all repos in the organization. Community members who wish to become members of the organization should meet the following requirements, which are open to the discretion of the steering committee:
```
Comment (leecalcote) [inline GOVERNANCE.md:9]: Revert this. It was correct previously.

File: GOVERNANCE.md [hunk 3]
```diff
@@ -25,11 +25,11 @@ The Meshery project consists of multiple repositories. Each repository is subjec
...
+Maintainers are individuals that have demonstrated their dedication the betterment of the project and dedication to Maintainers are empowered to review, approve, merge, close issues and pull requests. Maintainers are empowered to make releases. These individuals are most experience with the given project (or specific repo) and are expected to lead its growth and improvement. Adding and removing maintainers for a given repo is the responsibility of the existing maintainer team for that repo and therefore does not require approval from the steering committee.
 
-This privilege is granted with some expectation of responsibility: maintainers are people who care about the Meshery project and want to help it grow and improve.
+This privilege is granted with some expectation of responsibility: maintainers are people who care about the Meshery project and want to help it grows and improve.
```
Comment (leecalcote) [inline GOVERNANCE.md:32]: Revert this. It was correct previously.

File: GOVERNANCE.md [hunk 4]
```diff
@@ -25,11 +25,11 @@ The Meshery project consists of multiple repositories. Each repository is subjec
...
 
-Maintainers are individuals that have demonstrated their dedication the betterment of the project and dedication to Maintainers are empowered to review, approve, merge, close issues and pull requests. Maintainers are empowerd to make releases. These individuals are most experience with the given project (or specific repo) and are expected to lead its growth and improvement. Adding and removing maintainers for a given repo is the responsibility of the existing maintainer team for that repo and therefore does not require approval from the steering committee.
+Maintainers are individuals that have demonstrated their dedication the betterment of the project and dedication to Maintainers are empowered to review, approve, merge, close issues and pull requests. Maintainers are empowered to make releases. These individuals are most experience with the given project (or specific repo) and are expected to lead its growth and improvement. Adding and removing maintainers for a given repo is the responsibility of the existing maintainer team for that repo and therefore does not require approval from the steering committee.
```
Comment (leecalcote) [inline GOVERNANCE.md:30]: Thanks!

**Comment by 404Nikhil** [comment]
I have committed the suggested changes.

**Review by leecalcote — APPROVED** [review]
Good to go, @404Nikhil 👍

**Pull request merged**